packages/file-upload/src/from-url/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { useState } from 'react';
+import { Box, TextField, Typography } from '@mui/material';
+import { FromURLContainer } from './styles';
+import { Button } from '../../../button/src/index';
+
+export interface FromUrlProps {
+  accept?: string;
+  acceptLabels?: string;
+  variant?: 'single' | 'combined';
+  componentDisabled?: boolean;
+  buttonDisabled?: boolean;
+  inputDisabled?: boolean;
+  onAccept?: (url: string) => void;
+  onReject?: (url: string) => void;
+  validator?: (url: string) => string | null;
+}
+
+export function FromUrl(props: FromUrlProps) {
+  const {
+    accept,
+    acceptLabels,
+    variant = 'single',
+    componentDisabled,
+    inputDisabled,
+    buttonDisabled,
+    onAccept = () => undefined,
+    onReject = () => undefined,
+    validator,
+  } = props;
+
+  const [url, setUrl] = useState('');
+  const [error, setError] = useState<string | null>(null);
+  const [loading, setLoading] = useState(false);
+
+  function isValidUrl(string) {
+    try {
+      new URL(string);
+      return true;
```

Comment:
```suggestion
     return Boolean(new URL(urlString)); 
```

Reply:
In this case, the try catch is needed since URL throws an error when the URL is invalid